assets/package.json

```diff
@@ -1,18 +1,11 @@
 {
-  "name": "@massds/mayflower-tokens",
-  "version": "1.4.2",
-  "description": "Mayflower design tokens — it includes color token SCSS variables, fonts, icons and other imagery",
-  "main": "dist/scss/index",
-  "style": "dist/scss/index",
+  "name": "@massds/assets",
```

Comment:
```suggestion
  "name": "@massds/mayflower-assets",
```

Reply:
Can you also fix the steps in circleci config? I think just change mayflower-assets to mayflower-tokens and remove the build step?